pydfy/models.py

```diff
@@ -51,6 +51,15 @@ class Number(Component):
     template_path: str = field(kw_only=True, default="src/partials/number.html")
 
 
+@dataclass
+class Kpi(Component):
+    current: Union[int, float]
+    goal: Union[int, float]
```

Comment:
I believe "target" is more commonly used

Reply:
Fair enough (I am unfamiliar)